moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -194,5 +194,7 @@ namespace moveit
 namespace planning_interface
 {
 using MoveItCpp[[deprecated("use moveit_cpp")]] = moveit_cpp::MoveItCpp;
+using MoveItCppPtr[[deprecated("use moveit_cpp")]] = moveit_cpp::MoveItCppPtr;
+using MoveItCppConstPtr[[deprecated("use moveit_cpp")]] = moveit_cpp::MoveItCppConstPtr;
```

Comment:
I guess you should add `MOVEIT_CLASS_FORWARD` explicitly, otherwise you miss some other types as well.

Reply:
good point, thx